_layouts/ballot.html

```diff
@@ -6,8 +6,12 @@
 
 {% assign today = site.time | date: "%Y-%m-%d" %}
 {% assign current_election = locality.current_election | date: "%Y-%m-%d" %}
+{% assign data_totals = site.data.totals[ballot.data_key] %}
```

Comment:
This should be updated to use the new var name introduced [here](bb2052a255b45b5756913fcf688e820c203a9535).

Reply:
fixed!